en/gps_compass/rtk_gps_locosys_r2.md

```diff
@@ -0,0 +1,109 @@
+# LOCOSYS Hawk R2 RTK-Compass GPS
+
+The [LOCOSYS Hawk R2](https://www.locosystech.com/en/product/hawk-r2.html) is a dual-frequency [RTK GPS module](../gps_compass/rtk_gps.md) receiver designed for compatibility with Pixhawk.
+Additionally, it is equipped with an e-compass.

```

Comment:
What is an "e-compass"? I've asked about compass part elsewhere. It might be that this is some kind of estimated compass from GPS? If so, does it require any special setup?

Reply:
it is the same mag IST8310. I do not know why they call it like this. Just confirmed with them and it seems like it is same.I will change all to magnetometer then. 
![Locosys_datasheet_Mag](https://user-images.githubusercontent.com/46557204/198997789-35e51c0f-a619-4a2d-bc1f-d305cf1bdcf1.png)